### DISCUSSION TITLE
Is there a way to draw color tolerance ellipse in ab plane?

### DISCUSSION THREAD

**Discussion by starkcolour** [colour-science/colour#903] [Q&A] [answered]
Color tolerance ellipse in ab plane(CIELab color space)  is very useful for analyse the color reproduction quality, especially in camera color characterization, usually we want to draw an ellipse(4x,maybe.) based on the reference data like Color checker, or any other customer data sets.   I went through sub-packages like plotting, found only MacAdam ellipse in CIE1931 or CIE1960UCS color spaces.
There's also a need for different  ellipse methods such as deltaE94 and deltaE00.
Thanks a lot

Pengfei

**Comment by thomasmansencal** [comment, marked as answer]
Hi @starkcolour,

Sorry we don't have such a feature out-of-the-box, arguably, it would be useful though!

Cheers,

Thomas

**Reply by starkcolour** [reply]
Thanks for the quick response, this saved my time for searching.
By the way, excellent work you've done in the color plotting sub-packages, I'm trying other features.
